comic.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+type Comic struct {
+	Num        int    `json:"num"`
+	SafeTitle  string `json:"safe_title"`
```

Comment:
I tried the underscore naming and it's an absolute menace when you try to work with it on the client, would much prefer it like ```safeTitle```